scripts/apps/authoring/metadata/PlacesService.ts

```diff
@@ -1,35 +1,64 @@
 import {omit} from 'lodash';
 
 interface IGeoname {
+    /** name of the place */
```

Comment:
it's just name of town like `Prague`, in the language requested, we don't use addresses there

Reply:
Are both `IGeoname` and `ILocated` only return city level info and not district/street? It would be good to mention that in the comment above the interface declaration.